### DISCUSSION TITLE
HLS Parameter environment in Docker

### DISCUSSION THREAD

**Discussion by yanhua1012** [bluenviron/mediamtx#1193] [Questions] [answered]
I start a linux container with version v0.20.0

I set envrionment as follows  
-e RTSP_PROTOCOLS=tcp
-e HLS_HLSADDRESS=:443
the rtsp parameter works, but the hls one not work.
the log show [HLS] listener opened on default port 8888

does environment  support hls parameter? thanks.

**Comment by aler9** [comment, marked as answer]
Hello, the correct flag is

```
-e RTSP_HLSADDRESS=:443
```

The `RTSP_` prefix must be added before every environment variable and is being kept for compatibility purposes. It was conceived way before the server started supporting HLS.

[a comment by an automated account is not shown]
